### PR TITLE
Remove fork of BusTopologies from rocket-chip

### PR DESCRIPTION
**Pull request by jerryz123**
**Type of change**:
- [ ] Bug fix
- [ ] New feature
- [x] Other enhancement

**Impact**:
- [x] Other

**Contributor Checklist**:
- [x] Did you set `main` as the base branch?
- [x] Did you delete any extraneous prints/debugging code?

**Release Notes**

- Remove the custom bus-topologies from Chipyard, we can now use the upstream ones in rocket-chip

### CODE REVIEW

File: generators/chipyard/src/main/scala/config/AbstractConfig.scala [hunk 1]
```diff
@@ -41,20 +41,21 @@ class AbstractConfig extends Config(
...
+  new chipyard.config.WithInheritBusFrequencyAssignments ++         // Unspecified clocks within a bus will receive the bus frequency if set
+  new chipyard.config.WithPeripheryBusFrequencyAsDefault ++         // Unspecified frequencies with match the pbus frequency (which is always set)
+  new chipyard.config.WithMemoryBusFrequency(100.0) ++              // Default 100 MHz mbus
+  new chipyard.config.WithPeripheryBusFrequency(100.0) ++           // Default 100 MHz pbus
+  new freechips.rocketchip.subsystem.WithJtagDTM ++                 // set the debug module to expose a JTAG port
+  new freechips.rocketchip.subsystem.WithNoMMIOPort ++              // no top-level MMIO master port (overrides default set in rocketchip)
+  new freechips.rocketchip.subsystem.WithNoSlavePort ++             // no top-level MMIO slave port (overrides default set in rocketchip)
+  new freechips.rocketchip.subsystem.WithInclusiveCache ++          // use Sifive L2 cache
+  new freechips.rocketchip.subsystem.WithNExtTopInterrupts(0) ++    // no external interrupts
+  new freechips.rocketchip.subsystem.WithDontDriveBusClocksFromSBus // leave the bus clocks undriven by sbus
```
Comment (abejgonzalez) [inline generators/chipyard/src/main/scala/config/AbstractConfig.scala:59]:
```suggestion
  new freechips.rocketchip.subsystem.WithDontDriveBusClocksFromSBus ++ // leave the bus clocks undriven by sbus
```

**Comment by jerryz123** [comment]
This needs https://github.com/chipsalliance/rocket-chip/pull/2940

**Comment by abejgonzalez** [comment]
This PR should be updated now correct?

**Comment by jerryz123** [comment]
Merged into #1216

**Pull request closed without merging**